packages/yourloops/test/integration/pages/trends/trends-view-common.spec.tsx

```diff
@@ -99,13 +111,14 @@ describe('Trends view for anyone', () => {
       renderPage('/trends')
 
       await checkReadingsInRangeStatsWidgets()
+
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches to further communicate your intent.  -->
**🎯 suggestion (non-blocking):** ‏line break from lines 113 to 118 could be removed here 😉 (not mandatory, it's just my strictness who takes over 😅)